src/rard/research/views/__init__.py

```diff
@@ -89,7 +89,7 @@
     fetch_fragments,
 )
 from .history import HistoryListView
-from .home import HomeView, render_editor_modal_template
```

Comment:
This doesn't ring a bell at all. Any idea what it was?

Reply:
that was because I had to do a modal for the editor for like footnotes